README.md

```diff
@@ -214,8 +214,8 @@ Example:
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/214269878-432b849f-fe9a-454e-a757-6f30e969743c.png)

**[Optional]**

- [ ] It would be nice if you add some most frequently asked questions and their answers in this section to guide your viewers about your project and its uniqueness and to make your documentation readme file descriptive :+1:

Reply:
Done as advised